apps/evm_family/evm_context.h

```diff
@@ -0,0 +1,59 @@
+/**
+ * @file    evm_context.h
+ * @author  Cypherock X1 Team
+ * @brief   Ethereum config for various network chains
+ * @copyright Copyright (c) 2023 HODL TECH PTE LTD
+ * <br/> You may obtain a copy of license at <a href="https://mitcc.org/"
+ * target=_blank>https://mitcc.org/</a>
+ */
+#ifndef EVM_CONTEXT_H
+#define EVM_CONTEXT_H
+
+/*****************************************************************************
+ * INCLUDES
+ *****************************************************************************/
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "evm_contracts.h"
+
+/*****************************************************************************
+ * MACROS AND DEFINES
+ *****************************************************************************/
+
+/// this makes length of 7 with a termination NULL byte
+#define EVM_SHORT_NAME_MAX_SIZE 8
+/// this makes length of 15 with a termination NULL byte
+#define EVM_LONG_NAME_MAX_SIZE 16
+
+/*****************************************************************************
+ * TYPEDEFS
+ *****************************************************************************/
+
+typedef struct {
+  /** Largest unit/denomination indicator/symbol. This will be used in UX for
+   * displaying fees and amount.
+   */
+  const char lunit_name[EVM_SHORT_NAME_MAX_SIZE];
```

Comment:
Would we also need the lunit conversion decimal value in this config?

Reply:
no. did not find it necessary. all the currently supported chains have the same decimal value (18). the decimal value varies in tokens only. ref: https://app.clickup.com/t/9002019994/PRF-4320